TodoList/TodoList.Api/Controllers/NodeController.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Net;
+using System.Threading.Tasks;
+using System.Web.Http;
+using TodoList.Api.Models;
+
+namespace TodoList.Api.Controllers
+{
+    public class NodeController : ApiController
+    {
+        public NodeModel[] Nodes { get; set; }
+
+        public NodeController()
+        {
+            Nodes = InitializeData();
+        }
+
+        [Route("api/v1/nodes")]
+        public async Task<IHttpActionResult> GetAsync()
+            => await Task.FromResult<IHttpActionResult>(Ok(Nodes));
+
+        [Route("api/v1/nodes/{id}")]
+        public async Task<IHttpActionResult> GetAsync(string id)
+            => await Task.FromResult<IHttpActionResult>(Ok(Nodes[0]));
+
+        [Route("api/v1/nodes/{text}")]
+        public async Task<IHttpActionResult> PostAsync(string text)
+            => await Task.FromResult<IHttpActionResult>(Content(HttpStatusCode.Created, Nodes[1]));
```

Comment:
Your `POST` action does not return location header (see `[rfc2616](https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.30)`)